src/main/webapp/WEB-INF/view/register.jsp

```diff
@@ -3,34 +3,30 @@
 <html>
 
 <head>
-
-    <title>Register</title>
-    <link rel="stylesheet" href="/css/main.css">
-
+  <title>The Unnamed Ones&#39; CodeU Chat App</title>
+  <link rel="stylesheet" href="/css/main.css">
     <style>
         label{
             display: inline-block;
             width 100px;
         }
     </style>
-
 </head>
 
 <body>
-
-    <nav>
-        <a id="navTitle" href="/">CodeU Chat App </a>
-        <a href="/conversations">Conversations</a>
-
-        <% if (request.getSession().getAttribute("user") != null){
-            %>
-
-            <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
-
-            <% }else{ %>
-                <a href="/login">Login</a>
-            <% } %>
-    </nav>
+   <nav>
+     <a id="navTitle" href="/">The Unnamed Ones&#39; CodeU Chat App</a>
+    
+     <% if(request.getSession().getAttribute("user") != null){ %>
```

Comment:
Hm. This actually hints that there are a couple bugs here: why would we allow someone who is logged in to use the /register page?  It seems like if they are logged in, it should automatically redirect to conversations, but then we also need a "log out" feature.  Not something you need to fix here, but maybe put a pin in that for later if you're looking for something to do.

Also, I wonder if you should pull this out into a second include, so that there are two flavors of include for the navbar.